examples/tools/tool-package-quickstart/my_tool_package/tools/tool_with_dynamic_list_input.py

```diff
@@ -0,0 +1,32 @@
+from promptflow import tool
+from typing import List, Union, Dict
+
+
+def dummy_list(prefix: str = "", size: int = 10, **kwargs) -> List[Dict[str, Union[str, int, float, list, Dict]]]:
+    # dummy random.
+    import random
+
+    words = ["apple", "banana", "cherry", "date", "elderberry", "fig", "grape", "honeydew", "kiwi", "lemon"]
+    result = []
+    for i in range(size):
+        random_word = random.choice(words)
```

Comment:
It is still possible to have duplicate numbers in the loop

Reply:
Why?

Reply:
change to unique values